checkly/resource_check.go

```diff
@@ -203,8 +203,9 @@ func resourceCheck() *schema.Resource {
 				Description: "The id of the runtime to use for this check.",
 			},
 			"alert_channel_subscription": {
-				Type:     schema.TypeList,
-				Optional: true,
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "An array of channel ID and if it's activated or not. If you don't set at least one alert subscription for your check, we won't be able to alert you in case something goes wrong with it.",
```

Comment:
We allow more than one channel ID, right? Does this make sense?
```suggestion
				Description: "An array of channel IDs and whether they're activated or not. If you don't set at least one alert subscription for your check, we won't be able to alert you in case something goes wrong with it.",
```